### PR TITLE
Functions called before being defined

### PR DESCRIPTION
**Pull request by okwolf**
[Hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) is awesome and everything, but for me code is easier to read when functions are defined before being called. I also have this weird thing about putting `export`s at the end of my files.

Feel free to reject this if it's too nitpicky.

### CODE REVIEW

**Comment by JorgeBucaran** [comment]
@okwolf +3 bytes.

**Comment by JorgeBucaran** [comment]
@okwolf I'll merge this and rearrange and tweak a bit later to my taste.

File: test/dom.test.js [hunk 1]
```diff
@@ -1,5 +1,38 @@
 import { h, app } from "../src"
 
+function testTreeSegue(name, trees) {
```
Comment (JorgeBucaran) [inline test/dom.test.js]: @okwolf What's happening here?

Comment (okwolf) [reply test/dom.test.js]: @JorgeBucaran I'm not familiar with this function, all I did was move it from the bottom of the file up toward to top.

**Comment by okwolf** [comment]
> @JorgeBucaran +3 bytes.

I'm a bit surprised the size changed from reordering the existing code. Interesting 🤔

**Pull request merged**